Workbooks/Azure Web Application Firewall/WAF monitoring/WAF Monitoring.workbook

```diff
@@ -0,0 +1,831 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Sandy Corp Organization Events",
```

Comment:
this is even right at the top, all customers would have this same title, is this really what you want?

Reply:
I have changed this to be Azure WAF Monitoring metrics

Reply:
the word "Sandy" appears all over the place, in queries like 
```
| where policy_s == "sandycookieWAFpolicy" or policy_s  == "AmysAPIMwaf" or policy_s =="sandyappgwdcWAFpolicy"
```

in some queries it is commented out, in some it is not?

is sandy a feature name or something i don't know about or is this specific to your sample data and won't work anywhere else?